libs/components/packages/src/schematics/rules/workspace-check/workspace-check.spec.ts

```diff
@@ -14,8 +14,14 @@ describe('Workspace check', () => {
   it('should warn when SSR is enabled', async () => {
     const tree = await createTestApp(runner, {
       projectName: 'test-project',
-      options: { ssr: true },
```

Comment:
Something changed in Angular 19 where the `ng-new` schematic won't complete its async process in the test if `ssr` is set. Setting the option directly in the angular.json serves our purposes and doesn't cause a timeout.